pyquil/tests/test_reference_density_simulator.py

```diff
@@ -1,12 +1,23 @@
 import numpy as np
 import pytest
+import itertools
+import functools
+from operator import mul
+import networkx as nx
+import logging
 
 import pyquil.gate_matrices as qmats
 from pyquil import Program
 from pyquil.gates import *
 from pyquil.pyqvm import PyQVM
 from pyquil.reference_simulator import ReferenceDensitySimulator, ReferenceWavefunctionSimulator
 from pyquil.unitary_tools import lifted_gate_matrix
+from pyquil.paulis import sI, sX, sY, sZ
+from pyquil.device import NxDevice
+from pyquil.api import QuantumComputer
+from pyquil.api._qac import AbstractCompiler
+from pyquil.operator_estimation import measure_observables, ExperimentSetting, \
```

Comment:
prefer the `from .. import (.., .., ..)` syntax over line breaks

Reply:
- [ ] todo